ee/clickhouse/migrations/0009_person_deleted_column.py

```diff
@@ -0,0 +1,9 @@
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.sql.person import KAFKA_PERSONS_TABLE_SQL
+
+operations = [
+    migrations.RunSQL("DROP TABLE kafka_person"),
+    migrations.RunSQL("ALTER TABLE person ADD COLUMN IF NOT EXISTS is_deleted Boolean DEFAULT 0"),
```

Comment:
Both of these need `ON CLUSTER posthog`

We also need an `ALTER TABLE` for `person_mv` to add this column as well so that it pulls this column from kafka and inserts it into `person` table